test/burn/org/apache/cassandra/net/ConnectionBurnTest.java

```diff
@@ -256,19 +271,19 @@ List<Connection> getConnections(InetAddressAndPort endpoint, boolean inbound)
         /**
          * Test connections with broken messages, live in-flight bytes updates, reconnect
          */
-        public void run() throws ExecutionException, InterruptedException, NoSuchFieldException, IllegalAccessException, TimeoutException
+        public void run() throws Exception
         {
             Reporters reporters = new Reporters(endpoints, connections);
+            List<Exception> exceptions = new LinkedList<>();
```

Comment:
why a linked list?

Reply:
also, consider declaring more proximal to usage

Reply:
I meant that `LinkedList` is almost always inferior to `ArrayList`, and there appears to be no reason to prefer it here. It's not an important point, particularly here the distinction is unimportant.

Regarding declaring it more proximal to usage, I meant I don't see why it is declared at the start of the method rather than above the loop that uses it. It's helpful IMO to declare variables close to where they are logically introduced in the code.